edegens/ptype
# inital optimus

## What this pr does
basically what we demoed to franks but it actually works now. I just couldn't think on an empty stomach.

### How to test
1. `cd` into coordinator dir and `./run`
2. `cd` into worker dir and `./run` and `./run2`
3. send a post request to `localhost:8082/test` with param `target` equal to whatever number u wanna check.

It should return a number that `target` is divisible by, if `target` is returned then it is prime!
